modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -156,13 +157,24 @@ struct GAPI_EXPORTS_W_SIMPLE OpenVINO {
 
     Constructs OpenVINO parameters based on device type information.
 
-    @param dev_type Target device type to use. ("CPU_FP32", "GPU_FP16", etc)
+    @param dev_type Target device type to use. ("CPU", "GPU", "GPU.0" etc)
     */
     GAPI_WRAP
     explicit OpenVINO(const std::string &dev_type)
         : device_type(dev_type) {
     }
 
+    /** @brief Class constructor.
+
+    Constructs OpenVINO parameters based on map of options passed.
+
+    * @param params A map of parameter names and their corresponding string values.
+    */
+    GAPI_WRAP
+    explicit OpenVINO(const std::map<std::string, std::string>& params)
+        : params_map(params), is_initialized_from_map(true) {
```

Comment:
Do we need `is_initialized_from_map` field? Can we just check if `params_map` empty or not?

Reply:
Alright! Makes sense. Changed